src/NServiceBus.RabbitMQ/DelayedDelivery/DelayInfrastructure.cs

```diff
@@ -73,13 +75,22 @@ public static void TearDown(IModel channel)
             }
         }
 
-        public static StartupCheckResult CheckForInvalidSetting(bool routingTopologySupportsDelayedDelivery, bool disableTimeoutManager)
+        public static StartupCheckResult CheckForInvalidSettings(SettingsHolder settings)
```

Comment:
This because I know this will run after feature setup, so I can check for the final state of the timeout manager feature when this is called. Doing it earlier would end up being before feature setup, and prerequisites are evaluated as part of feature setup.